complaint_search/es_interface.py

```diff
@@ -18,26 +24,56 @@
 _COMPLAINT_ES_INDEX = os.environ.get('COMPLAINT_ES_INDEX', 'complaint-index')
 _COMPLAINT_DOC_TYPE = os.environ.get('COMPLAINT_DOC_TYPE', 'complaint-doctype')
 
-def get_es():
+def _get_es():
     global _ES_INSTANCE
     if _ES_INSTANCE is None:
         _ES_INSTANCE = Elasticsearch([_ES_URL], http_auth=(_ES_USER, _ES_PASSWORD), 
             timeout=100)
     return _ES_INSTANCE
 
-def get_meta():
+def _get_now():
+    return datetime.now()
+
+def _four_business_days_ago():
+    # show notification starting fifth business day data has not been updated
```

Comment:
1. I like calling out the logic into a separate function that can be updated when the business rules change
1. What if the rule changes to 3 business days?  Maybe something that doesn't embed the logic in the name